TestFoundation/TestNSURLSession.swift

```diff
@@ -101,13 +158,7 @@ class TestURLSession : XCTestCase {
             }
             let httpResponse = response as! HTTPURLResponse?
             XCTAssertEqual(200, httpResponse!.statusCode, "HTTP response code is not 200")
-            do {
-                let json = try JSONSerialization.jsonObject(with: data!, options: [])
-                let arr = json as? Array<Any>
-                let first = arr![0]
-                let result = first as? [String : Any]
-                expectedResult = result!["capital"] as! String
-            } catch { }
+            expectedResult = NSString(data: data!, encoding: String.Encoding.utf8.rawValue)!._bridgeToSwift()
```

Comment:
I believe we shouldn't be using the `_bridgeToSwift` function outside of Foundation's implementation itself.